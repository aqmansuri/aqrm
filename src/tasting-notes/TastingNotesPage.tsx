import {
  IonPage,
  IonHeader,
  IonTitle,
  IonToolbar,
  IonContent,
} from '@ionic/react';

const TastingNotesPage: React.FC = () => {
  return (
    <IonPage>
      <IonHeader>
        <IonToolbar>
          <IonTitle>Tasting Notes</IonTitle>
        </IonToolbar>
      </IonHeader>
      <IonContent>
        <IonHeader collapse="condense">
          <IonToolbar>
            <IonTitle size="large">Tasting Notes</IonTitle>
          </IonToolbar>
        </IonHeader>
      </IonContent>
    </IonPage>
  );
};
export default TastingNotesPage;
