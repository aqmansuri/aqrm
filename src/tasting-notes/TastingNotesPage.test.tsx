import { render } from '@testing-library/react';
import TastingNotesPage from './TastingNotesPage';

describe('<TastingNotesPage />', () => {
  it('renders consistently', () => {
    const { asFragment } = render(<TastingNotesPage />);
    expect(asFragment).toMatchSnapshot();
  });
});
