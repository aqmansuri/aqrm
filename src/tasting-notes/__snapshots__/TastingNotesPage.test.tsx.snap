// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TastingNotesPage /> renders consistently 1`] = `[Function]`;
